.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+### General (all pull requests)
+
+- [ ] The changes follow the [formatting](https://github.com/ppy/osu-wiki/tree/master/wiki/Article_styling_criteria/Formatting) and [writing](https://github.com/ppy/osu-wiki/tree/master/wiki/Article_styling_criteria/Writing) style criteria.
+- [ ] There is an existing relevant GitHub issue (specify which one).
```

Comment:
i don't think this one is actually needed

it might also scare away potential contributors?

Reply:
good point